libs/sync/src/lib/sync/new-sync.service.ts

```diff
@@ -0,0 +1,476 @@
+        description: 'Microsoft OneDrive file hosting service',
+        permanentSyncOnly: true,
+        factory: (settings) => of(new OAuthConnector('onedrive', settings?.['id'] || '', this.config.getAPIOrigin())),
+      },
+      settings: {},
+    },
+    sharepoint: {
+      definition: {
+        id: 'sharepoint',
+        title: 'SharePoint',
+        logo: `${baseLogoPath}/sharepoint.svg`,
+        description: 'Microsoft Sharepoint service',
+        permanentSyncOnly: true,
+        factory: (settings) => of(new SharepointImpl('sharepoint', settings?.['id'] || '', this.config.getAPIOrigin())),
+      },
+      settings: {},
+    },
+    dropbox: {
+      definition: {
+        id: 'dropbox',
+        title: 'Dropbox',
+        logo: `${baseLogoPath}/dropbox.svg`,
+        description: 'File storage and synchronization service developed by Dropbox',
+        permanentSyncOnly: true,
+        factory: (settings) => of(new OAuthConnector('dropbox', settings?.['id'] || '', this.config.getAPIOrigin())),
+      },
+      settings: {},
+    },
+    folder: { definition: FolderConnector, settings: {} },
+    sitemap: { definition: SitemapConnector, settings: {} },
+    confluence: { definition: ConfluenceConnector, settings: {} },
+    // s3: { definition: S3Connector, settings: {} },
+    // gcs: { definition: GCSConnector, settings: {} },
+    // brightcove: { definition: BrightcoveConnector, settings: {} },
+  };
+  destinations: { [id: string]: { definition: DestinationConnectorDefinition; settings: ConnectorSettings } } = {
+    nucliacloud: {
+      definition: NucliaCloudKB,
+      settings: {
+        backend: this.config.getAPIURL(),
+        client: this.config.staticConf.client,
+      },
+    },
+  };
+  sourceObs = new BehaviorSubject(Object.values(this.sources).map((obj) => obj.definition));
+  private _syncServer = new BehaviorSubject<string>(localStorage.getItem(SYNC_SERVER_KEY) || '');
+  syncServer = this._syncServer.asObservable();
+
+  private _step = new BehaviorSubject<number>(-1);
+  step = this._step.asObservable();
+  showSource = new Subject<{ connectorId: string; sourceId: string }>();
+  addSource = new Subject<void>();
+  private _isServerDown = new BehaviorSubject<boolean>(true);
+  isServerDown = this._isServerDown.asObservable();
+  private _currentSourceId = new BehaviorSubject<string | null>(null);
+  currentSourceId = this._currentSourceId.asObservable();
+  private _sourcesCache = new BehaviorSubject<{ [id: string]: Source }>({});
+  sourcesCache = this._sourcesCache.asObservable();
+  currentSource = combineLatest([this.sourcesCache, this.currentSourceId]).pipe(
+    map(([sources, sourceId]) => sources[sourceId || '']),
+  );
+  private _basePath = new BehaviorSubject<string>('/');
+  basePath = this._basePath.asObservable();
+
+  constructor(
+    private sdk: SDKService,
+    private http: HttpClient,
+    private config: BackendConfigurationService,
+  ) {
+    console.log('NewSyncService');
```

Comment:
console.log

Reply:
yes I keep it to make sure I am using the temporary new service